test/browser/clubs-page.test.jsx

```diff
@@ -97,6 +97,7 @@ describe("ClubsPage.ClubList", function() {
     owner: 'foo',
     website: 'http://example.org',
     location: 'Somewhere, USA',
+    status: 'approved',
     name: 'foo club'
   }];
 
```

Comment:
(Github didn't let me make a comment at [line 104](https://github.com/mozilla/teach.webmaker.org/pull/965/files#diff-23085ec9521eccdf034ed187c1843e06R104), there was no "+" when I hovered over)

Maybe we can also check in the test below to see if any `<div>` with classes `.alert.alert-warning ` or `.alert.alert-danger` is presented?